docs/General/Manuals/client-manual.rst

```diff
@@ -311,6 +311,51 @@ scenario file in an alternate location. Clicking on :guilabel:`Cancel` will brin
 Options
 -------
 
+When you click on the :guilabel:`Options` button, a dialog box will appear as in Figure 8 below. This dialog
+allows you to set a myriad of options that effect the look at feel of the client when you are playing a game.
```

Comment:
~effect~ affect the look or ~effect~ change the look, effect the look sounds wrong